test/template-tests/testcases/CreateUIDefinition/Textboxes-Are-Well-Formed.test.ps1

```diff
@@ -38,7 +16,8 @@ foreach ($textbox in $allTextBoxes) {
         if (-not $textbox.constraints.validationMessage) {
             Write-Error "Textbox $($textbox.Name) is missing constraints.validationMessage"
         }
+        if (-not $textbox.constraints.maxLength) {

```

Comment:
I don't think this is needed - pw control should have a max based on the OS type?

Reply:
Wouldn't that be a separate test for password controls?  Or for textboxes used within Password controls?  Point me to the exception and I'll refine the rule.

Reply:
Ok, we can remove the maxLength check.